Ginger/Ginger/Variables/VariablesPage.xaml

```diff
@@ -1,20 +0,0 @@
-    <Page
```

Comment:
Why this class was deleted?

Reply:
Not sure, @Jinendra Can you please confirm?